internal_workflow_testsuite.go

```diff
@@ -1124,5 +1124,10 @@ func (env *testWorkflowEnvironmentImpl) signalWorkflow(name string, input interf
 	}, true)
 }
 
+func (env *testWorkflowEnvironmentImpl) GetVersion(changeID string, minSupported, maxSupported Version) Version {
+	// TODO: Add code that returns the same version for all calls for the given changeID.
+	return maxSupported
```

Comment:
always return maxSupported should be fine unless you expect workflow code to have multiple calls to GetVersion with same changeID but different minSupported/maxSupported. Is that even legit? What does it mean if user do that?